_episodes/03-func.md

```diff
@@ -151,16 +151,16 @@ Alt( [ 1 .. 10 ] )
 ~~~
 {: .output}
 
-The example above also demonstrates `time` - this is the variable which stores
+The example above also demonstrates `time` -- this is the variable which stores
```

Comment:
I am fine with `--` although would be nice to check how they are rendered in the html version.

Reply:
Actually, one example already exists in this lesson, I was just making this change for consistency.

See Chapter 1, the paragraph starting
```
Now let us consider an interesting calculation -- the average order of elements
```
It's a `--` in the markdown, and seems to be rendered as an en-dash (–).  So hopefully this will be okay!

Reply:
Perfect then!